package.json

```diff
@@ -64,6 +64,7 @@
     "jsx-control-statements": "^3.2.8",
     "lazysizes": "^4.1.6",
     "lodash": "4.17.15",
+    "node-sass": "^4.12.0",
```

Comment:
Why do we need to update this?

Reply:
@thestephenmarshall should i just delete that line?  I think that might have changed when I was going through the common errors page.